src/yadism/coefficient_functions/light/f3_nc.py

```diff
@@ -17,13 +22,25 @@ def NLO():
 
     def NNLO(self):
         """
-        |ref| implements :eqref:`4.8`, :cite:`vogt-f2nc`.
+        |ref| implements the difference between :eqref:`2.8` and :eqref:`3.5, :cite:`Davies:2016ruz`.
```

Comment:
This should be the correct reference for this coefficient function, but I'm a bit confused why this contribution is not directly added in `f3_cc.NonSingletEven` only.